airflow/www/utils.py

```diff
@@ -389,6 +393,17 @@ def __call__(self, field, **kwargs):
         return wtforms.widgets.core.HTMLString(html)
 
 
+def flask_admin_unescape(value):
+    """
+    Function to back out the CHAR_ESCAPE, CHAR_SEPARATOR values created by flask_admin
+    :param value: flask_admin id or ids to unescape
+    :return: unicode
+    """
+    return (as_unicode(value)
+            .replace(CHAR_ESCAPE + CHAR_ESCAPE, CHAR_ESCAPE)
+            .replace(CHAR_ESCAPE + CHAR_SEPARATOR, CHAR_SEPARATOR))
```

Comment:
Is there not a function in flask admin that will do it for us?

Reply:
Not that I was able to find. I did try to find one using the CHAR_ESCAPE value in the package's code but was unable to find anything: https://github.com/flask-admin/flask-admin/search?utf8=%E2%9C%93&q=CHAR_ESCAPE&type=